kcp/README.MD

```diff
@@ -1,22 +1,17 @@
 # Pipeline Services on managed KCP
 
 ## Clusters access
-The `login.sh -e staging` script will log you in into the various systems:
-1. KCP
-2. KCP workspace
-3. pipeline cluster
-4. ArgoCD
+Create a directory with the following files:
+* `argocd.yaml`: kubeconfig with the active context connecting to the argocd instance;
+* `kcp.yaml`: kubeconfig with the active context connecting to your KCP workspace;
+* `plnsvc.yaml`: the kubeconfig with with the active context connecting to the plnsvc cluster.
```

Comment:
This is assuming that there is a single workload cluster. We need to have something generic that can work with any number of workload clusters. Also, this is assuming that the reader knows what plnsvc cluster is but it is not explained.

Reply:
I'm documenting how it currently works. It certainly has short comings at the moment. The work we're currently doing should take care of handling multiple clusters.

I've replaced `plnsvc` by the full name.